projectsmember/Login_User.aspx.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Recaptcha.Web.UI.Controls;
+using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Web;
+using System.Web.UI;
+using System.Web.UI.WebControls;
+
+namespace projectsmember
+{
+    public partial class Login_User : System.Web.UI.Page
+    {
+        SqlConnection LOGConn = new SqlConnection(ConfigurationManager.ConnectionStrings["sojaIrishConnectionString"].ConnectionString);
+
+        protected void Page_Load(object sender, EventArgs e)
+        {
+            txtUserName.Attributes.Add("placeholder", "Username");
```

Comment:
Why don't we set these in client side? In html...
https://www.w3schools.com/tags/att_input_placeholder.asp

Reply:
You're Right. Resolved